bee-ledger/src/workers/pruning/batch.rs

```diff
@@ -16,11 +16,6 @@ use crate::{
 use bee_message::{
     milestone::{Milestone, MilestoneIndex},
     output::OutputId,
-    payload::{
-        indexation::{IndexationPayload, PaddedIndex},
```

Comment:
Is all the storage related functionality deleted because the new indexer will take those over?

Reply:
Indeed, it's not longer part of the core API. Since it'll be done through SQL, it's completely removed from KV.